tools/bin/commands/util/openshift_funcs

```diff
@@ -1,11 +1,66 @@
 #!/bin/bash
 
+OC_MIN_VERSION=3.9.0
+
+compare_version_part() {
+    local test=$1
+    local min=$2
+
+    test=$(($test + 0))
+    min=$(($min + 0))
+
+    if [ $test -eq $min ]; then
+        return 0;
+    elif [ $test -gt $min ]; then
+        return 1;
+    else
+        return 2;
+    fi
+}
+
+check_oc_version()
+{
+    set +e
```

Comment:
Please don't do this over the scope of a whole function, only in rare circumstances.